src/walter/output_manager.py

```diff
@@ -0,0 +1,580 @@
+
+
+class PeraxesLog(Log):
+    """
+    Log Peraxes throught time
+    """
+    fields = ["Init_flag", "Elapsed_time", "DOY", "Temperature", "Temp_cum", "Num_plante", "Num_talle", "Sheath_max",
+                 "Collar_height", "Dead_collar_height", "Delta_H", "Visible_leaves_surface", "Visible_surface",
+                 "Leaf_contrib_to_GAI", "Stem_contrib_to_GAI", "Ear_contrib_to_GAI", "Peduncle_contrib_to_GAI",
+                 "Stop_growth_flag"]
+
+    name = 'Peraxes'
+
+    def __init__(self):
+        """
+        Initialize the Peraxes output file
+        """
+        super(PeraxesLog, self).__init__()
+
+
+class DebugLog(Log):
+        """
+        Log Debug throught time
+        """
+        fields = ["Elapsed_time", "Temp_cum", "Num_plante", "Num_talle",
+                  "Organ_PAR",
+                  "Organ_type", "Num_organe", "Organ_surface", "Ei"]
+
+        name = 'Debug'
+        
+        def __init__(self):
+            """
+            Initialize the Debug output file
+            """
+            super(DebugLog, self).__init__()
+
+
+class ParPerAxesLog(Log):
+    """
+    Log ParPerAxes throught time
+    """
+    fields = ["Elapsed_time", "Temp_cum", "Num_plante", "Num_talle",
+              "Sum_PAR", "Inc_PAR"]
+    name = 'ParPerAxes'
+
+    def __init__(self):
+        """
+        Initialize the ParPerAxes output file
+        """
+        super(ParPerAxesLog, self).__init__()
+
+### --- Log --- ###
+
+"""
+#Comparaison des differentes signatures 
+
+log_Apex_Sirius    (day, current_temperature, _Tempcum, duree_du_jour, latitude, DOY, StA)
+log_Apex           (elapsed_time, Temperature, Tempcum, current_PAR, cohort_number, StA)
+log_Apex_R         (elapsed_time, DOY, Temperature, Tempcum, cohort_number, dico_stades, StAR)
+log_Proba          (elapsed_time, Temperature, Tempcum, GAI_prox, StBu)
+log_Blade          (elapsed_time, Tempcum, Temperature, cohort_number, leaf_emergence, StBl):
+log_Sheath         (elapsed_time, Tempcum, Temperature, cohort_number, StS):
+log_Ear            (elapsed_time, Tempcum, StE):
+log_Internode      (elapsed_time, Tempcum, Temperature, cohort_number, StI)
+log_Peduncle       (elapsed_time, Tempcum, Temperature, cohort_number, StP):
+log_Peraxes        (initialization_Flag, num_plt, elapsed_time, DOY, Temperature,Tempcum, axis, Sh_max_temp, Hcol_max, Hcol_dead, delta_H, tiller_surface, axis_census, leaf_contrib,
+                stem_contrib, ear_contrib, peduncle_contrib)
+log_Debug_PAR      (elapsed_time, Temperature, lstring):
+log_Gai            (initialization_Flag, num, elapsed_time, Tempcum, DOY, surface_plante, surface_plante_visible,
+            crop_scheme, GAI_tot, GAI_center, GAI_ind, GAI_prox):
+log_PAR_per_axes   (elapsed_time, Tempcum, num_plt, axis, current_PAR, dico_PAR_per_axis):
+"""
+
+def log_Apex_Sirius(Apex_Sirius_dico_df, day, current_temperature, _Tempcum, duree_du_jour, latitude, DOY, StA):
+    apex = Apex_Sirius_dico_df
+
+
```

Comment:
def log_Apex_Sirius(...):
  logger = get_global_logger()
  logger.log(name='ApexSirius', 
                    Elapsed_time=day, 
                    Temperature=current_temperature, ...)

Reply:
_LOGGER = None
def get_global_logger():
    if _LOGGER is None:
        global _LOGGER
        _LOGGER = Logger()
    return _LOGGER